src/equations/compressible_euler_3d.jl

```diff
@@ -248,7 +248,7 @@ in combination with [`source_terms_eoc_test_coupled_euler_gravity`](@ref)
 or [`source_terms_eoc_test_euler`](@ref).
 """
 function initial_condition_eoc_test_coupled_euler_gravity(x, t, equations::CompressibleEulerEquations3D)
-  # OBS! this assumes that γ = 2 other manufactured source terms are incorrect
+  # OBS! this assumes that gamma = 2 other manufactured source terms are incorrect
```

Comment:
Please, do not change this stuff here. It's definitely okay to use unicode in comments to make them more readable (but some people insisted on not using unicode for code, see the style guide).

Reply:
Sorry about that - search/replace caught that unintentionally.

Reply:
I think I accidentally modified some code in `@inline function source_terms_convergence_test(u, x, t, equations::CompressibleEulerEquations3D)` too. Reverting...

Reply:
You could just `get revert e8876197d238631eb625dee7778a29441139a9e1` and adapt the code for `entropy2cons` afterwards, e.g. based on my suggestion above.